src/style-spec/reference/v8.json

```diff
@@ -6016,6 +6019,9 @@
         0
       ],
       "length": 2,
+      "units": "degrees",
+      "minimum": 0,
+      "maximum": 90,
```

Comment:
Realizing that polar ranges 0-90 but azimuthal doesn't have the same constraint. Is there a way to specify the min/max of each number in the array, or would it be better to remove this altogether?

Reply:
The altitude ranges in [0,180] (0 at top/zenith, 180 below) while the azimuth in [0,360] (0 north and increasing clock-wise). I don't think our spec allows to untie these to only accept a range for one of the value but not the other, but I might be wrong.  This spec is matching to the light "position" spec, which does not have min and max. 

Unless we can assign separate values I'm in favor of removing.

Reply:
@thefifthisa did you have a path forward for this? We could add validation of multiple min/max values here  https://github.com/mapbox/mapbox-gl-js/blob/main/src/style-spec/validate/validate_number.js#L20, if not done in this PR, we can remove these values for the time being and address support for that separately (Otherwise we'd be constraining with an incorrect range).